target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
I got partial success : 


```
Starting file transfer ...........................................................................................................................OK (3932228 bytes received)
Checking new image ... OK

Your current software release is 42.2.
Loaded new image has release 1.0.

Now choose from the following:

(u) Update Flash ROM
(r) Write image to RAM and start it
(e) Exit

Enter (u, r or e): r

Booting BOSS...
boss image started at 0x5fb0034


OpenWrt kernel loader for MIPS based SoC
Copyright (C) 2011 Gabor Juhos <juhosg@openwrt.org>
Decompressing kernel... done!
Starting kernel at 80002000...

[    0.000000] Memory: 122972K/131072K available (4019K kernel code, 168K rwdata, 1172K rodata, 1216K init, 222K bss, 8100K reserved, 0K cma[    0.655520] gptu: succeeded to request irq 127tersr: 1, 8192 bytes)) bytes)ef]]idgeff]: 7645041785100000 ns
```

At this point, I have two issues:

- Serial port initialization issue
- Watchdog reset (this can be due to fail mouting rootfs).